package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "watt",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
In my opinion we should increase "patch" version instead of "minor".

Reply:
Ok, changed to 0.7.1